soc/arm/st_stm32/stm32g4/Kconfig.series

```diff
@@ -14,5 +14,6 @@ config SOC_SERIES_STM32G4X
 	select CPU_HAS_ARM_MPU
 	select CLOCK_CONTROL_STM32_CUBE if CLOCK_CONTROL
 	select HAS_PM
+	select HAS_SWO
```

Comment:
Out of curiosity, what is the use of `HAS_SWO`. I can't find any related code

Reply:
It's only used in `subsys/logging/backends/Kconfig.swo` as a dependency for `LOG_BACKEND_SWO`.

Reply:
Ok, my current working branch is not recent enough. I'll have a look once I'm done.